paypal/standard/forms.py

```diff
@@ -73,7 +73,7 @@ class PayPalPaymentsForm(forms.Form):
     DONATE = 'donate'
 
     # Where the money goes.
-    business = forms.CharField(widget=ValueHiddenInput(), initial=settings.PAYPAL_RECEIVER_EMAIL)
+    business = forms.CharField(widget=ValueHiddenInput())
```

Comment:
In addition to removing this, we need a backwards compatibility shim inside `PayPalPaymentsForm.__init__` that sets the `initial` value for this field to `settings.PAYPAL_RECEIVER_EMAIL`, if it exists, and raises a DeprecationWarning if it does exist.

This logic is probably complex enough to need a test to check it is being done. It's important that upgrading to 0.3.0 won't break anything for any users - they should only get a DeprecationWarning.